clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -132,6 +133,7 @@ func (m *clusterOOMsTrackerMeasurement) start(c clientset.Interface) error {
 	if err := informer.StartAndSync(i, m.stopCh, informerTimeout); err != nil {
 		return fmt.Errorf("problem with OOM events informer starting: %w", err)
 	}
+	m.startTime = time.Now()
```

Comment:
could you move it to initFields under m.isRunning = true ?
I think both should be initialized in the same place.

Reply:
Makes sense; done.